.pipelines/pipeline.user.windows.buddy.yml

```diff
@@ -13,9 +13,4 @@ build:
       artifacts:
         - to: 'Artifacts'
           include:
-            - '**/build/libs/*.jar'
-            - '**/build/libs/*-sources.jar'
-            - '**/build/libs/*-javadoc.jar'
-            - '**/build/libs/*.pom'
-          exclude:
-            - 'buildSrc/**/*'
+            - 'agent/agent/build/libs/*.jar'
```

Comment:
@heyams

Reply:
this is fine without including pom, sources, and javadoc.. that's for maven publish.